### PR TITLE
Python Images - Add basic packages to build C Python dependencies

### PR DESCRIPTION
**Pull request by Mallear**
* Add `basic-build`, `libffi-dev` and `openldap-dev` packages as they are often needed to build dependencies
* Upgrade pip version

### CODE REVIEW

File: python/Dockerfile.tpl [hunk 1]
```diff
@@ -3,14 +3,12 @@ FROM python:{{PYTHON_VERSION}}-alpine3.7
 ARG CI_HELPER_VERSION
 
 RUN echo "Starting ..." && \
-    apk add --update -q --no-cache bash curl git make openssh-client rsync tzdata && \
+    apk add --update -q --no-cache bash curl git make openssh-client rsync tzdata build-base libffi-dev openldap-dev && \
```
Comment (mremi) [inline python/Dockerfile.tpl]: Can you sort the packages please and update the CHANGELOG.md?

Comment (Mallear) [reply python/Dockerfile.tpl]: Changelog updated and packages sorted !

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Versions
 ========
 
+2018-08-DEV
+-----------
+
+* Add `basic-build`, `libffi-dev` and `openldap-dev` packages in Python images
```
Comment (mremi) [inline CHANGELOG.md]: Can you move these 2 lines in `2018-07-DEV` and rename it to `2018-08-DEV` please?

Comment (Mallear) [reply CHANGELOG.md]: Changelog updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Versions
 * Add an up to date mime types definition file for the images that use AWS CLI
 * Install PEAR in the PHP 5.3 image
 * Update Alpine Version for sonar image: 3.8
+* Add `basic-build`, `libffi-dev` and `openldap-dev` packages in Python images
```
[the remark this answers is not in the document]
Comment (Mallear) [reply CHANGELOG.md]: Oh yes, changed, sorry for the typo

**Pull request merged**